example-app/package.json

```diff
@@ -18,14 +18,15 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
-    "@admin-bro/express": "^3.0.0-beta.3",
-    "@admin-bro/mongoose": "^1.0.0-beta.3",
-    "@admin-bro/nestjs": "^1.0.0-beta.2",
```

Comment:
Can you remove it for sure?

Reply:
Yes, it was used in previous version in example-app, but it's silly to link lib from npm for example app that is created for that exact lib.